PRINCIPLES.md

```diff
@@ -60,6 +66,9 @@ These principles were derived from modern software operations but are rooted in
     One or more Administrators who are responsible for operating the runtime environments ie. installing, starting, stopping and updating software, code, configuration, etc.
     A set of policies controlling access and management of repositories, deployments, runtimes.
 
-- #### State Store
+- ### State Store
 
-    A system for storing versioned, immutable Desired States that provides access control and auditing on the changes to the Desired State. Git may be configured as a State Store, but [special precautions must be taken](recipes/SETTING_UP_GIT.md).
+    A system for storing immutable versions of _Desired State_ declarations.
+    This state store should provide access control and auditing on the changes to the Desired State.
+    Git is most often used as this State Store, but other systems may be used.
```

Comment:
should this be framed as "git is an example of a state store"?

Reply:
What about "Git has been the most common example of a system for storing desired state, but any other system that meets the above criteria when configured properly may be used"?

Reply:
fixed!